README.md

```diff
@@ -34,67 +34,40 @@ If you turn the [Doom logo upside down](https://www.reddit.com/r/Doom/comments/8
  * Integration of the NanoBSP node builder (enforce with `-bsp`) for maps without nodes or with unsupported nodes (not demo compatible).
- * Support for tall textures and sprites in DeePsea format has been added.
- * Support for unlimited extra states, sprites, mobjtypes and sounds has been added for use in Dehacked patches (supporting the "DEHEXTRA" and "DSDHacked" specs).
- * Support for changing in-game music using the "MUSINFO" lump has been added.
- * Support for the "UMAPINFO" lump has been added, compliant to Rev 2.2 of the [spec](https://github.com/kraflab/umapinfo).
- * Support for the "MBF21" compatibility level has been added, compliant to Rev 1.4 of the [spec](https://github.com/kraflab/mbf21).
- * Support for SMMU swirling animated flats.
- * Ability to customize the appearance of the extended Boom HUD using the [WOOFHUD](https://github.com/fabiangreffrath/woof/blob/master/docs/woofhud.md) lump.
-
-## Compatibility
-
- * All IWAD files since Doom 1.2 are supported, including Doom Shareware, The Ultimate Doom, Doom 2 and Final Doom. Furthermore, the IWAD files shipped with the "Doom 3: BFG Edition" and the ones published by the Freedoom project as well as Chex Quest, HACX and REKKR are supported.
- * UMAPINFO lumps for MASTERLEVELS.WAD, NERVE.WAD, SIGIL_V1_21.WAD, E1M4B.WAD and E1M8B.WAD have been added which are meant to be autoloaded with the corresponding PWAD and change map title, level transitions, par times, music and skies accordingly.
- * The concept of compatibility levels has been added, currently offering "Vanilla", "Boom", "MBF" and "MBF21" (default). The default compatibility level may be changed through the menu and overridden with the `-complevel` parameter, allowing for both numeric and named arguments, or the `COMPLVL` lump. Menu items in the Setup menu that don't apply to the current compatibility level are disabled and grayed out.
- * The project strives for full demo compatibility for all supported complevels. Furthermore, this port offers quality-of-life features like e.g. a demo progress bar, demo playback warp and skip as well as continued demo recording.
- * The SPECHITS, REJECT, INTERCEPTS and DONUT overflow emulations have been ported over from Chocolate Doom / PrBoom+, allowing for some more obscure Vanilla demos to keep sync.
+ * Tall textures and sprites in DeePsea format.
+ * Unlimited extra states, sprites, mobjtypes and sounds for use in Dehacked patches (supporting the "DEHEXTRA" and "DSDHacked" specs).
+ * In-game music changing using MUSINFO.
+ * UMAPINFO support, compliant to Rev 2.2 of the [spec](https://github.com/kraflab/umapinfo).
+ * MBF21 compatibility level, compliant to Rev 1.4 of the [spec](https://github.com/kraflab/mbf21).
+ * SMMU-style swirling animated flats.
+ * Customization the extended Boom HUD using the [WOOFHUD](https://github.com/fabiangreffrath/woof/blob/master/docs/woofhud.md) lump.
```

Comment:
The wording is a bit off here: I assume you wanted to write "Customization **_of_** the [...]".

Reply:
Yes, right. Sometimes the exact same sentence seems to read differently in a code editor and in a browser. 🤔